products/amazon-neptune.md

```diff
@@ -0,0 +1,51 @@
+---
+
+title: Amazon-Neptune
+category: db
+tags: amazon
+permalink: https://docs.aws.amazon.com/neptune/latest/userguide/engine-releases.html
+auto:
+-   custom: true
```

Comment:
Since we currently don't have custom automation for AWS Neptune, are you planning to work on this? (See [eks.py](https://github.com/endoflife-date/release-data/blob/main/src/eks.py) for eg).

We not, we can drop the `auto` key for now, till we add this.
```suggestion
```